src/test/scala/com/github/matsluni/akkahttpspi/BaseAwsClientTest.scala

```diff
@@ -52,21 +52,21 @@ trait LocalstackBaseAwsClientTest[C <: SdkClient] extends BaseAwsClientTest[C] {
 
   lazy val exposedServicePort: Int = 4566
 
-  override lazy val container: GenericContainer =
-    new GenericContainer(
-      dockerImage = "localstack/localstack",
-      exposedPorts = Seq(exposedServicePort),
-      env = Map("SERVICES" -> service),
-      waitStrategy = Some(LocalStackReadyLogWaitStrategy)
-    )
+  private lazy val containerInstance = new GenericContainer(
+    dockerImage = "localstack/localstack",
+    exposedPorts = Seq(exposedServicePort),
+    env = Map("SERVICES" -> service),
+    waitStrategy = Some(LocalStackReadyLogWaitStrategy)
+  )
+  override val container: GenericContainer = containerInstance
 }
 
 trait ElasticMQSQSBaseAwsClientTest[C <: SdkClient] extends BaseAwsClientTest[C] {
   def service: String
 
   lazy val exposedServicePort: Int = 9324
 
-  override lazy val container: GenericContainer =
```

Comment:
Scala3 does not like to override a `val` from a trait with a `lazy` declaration.

Reply:
Thanks for the explanation.